q2_types/feature_data/_format.py

```diff
@@ -149,18 +150,33 @@ def _validate_lines(self, max_lines):
                     return
                 if first[0] != ord(b'>'):
                     raise ValidationError("First line of file is not a valid "
-                                          "FASTA ID. FASTA IDs must start "
-                                          "with '>'")
+                                          "refline. reflines must start with "
+                                          "'>'")
                 fh.seek(0)
                 for line_number, line in enumerate(fh, 1):
                     if line_number >= max_lines:
                         return
                     line = line.decode('utf-8-sig')
                     if line.startswith('>'):
                         if last_line_was_ID:
-                            raise ValidationError('Multiple consecutive IDs '
-                                                  'starting on line '
+                            raise ValidationError('Multiple consecutive '
+                                                  'reflines starting on line '
                                                   f'{line_number-1!r}')
+                        line = line.split()
+                        if line[0] == '>':
+                            if len(line) == 1:
+                                raise ValidationError(
+                                    f'Refline on line {line_number} is '
+                                    'missing an ID.')
```

Comment:
Maybe also add `... missing an ID (or the ID starts with a space).` as I suspect it's more common to have a malformatted ID with no description segment, then a description with no ID.

Reply:
These are of course are indistinguishable situations.

Reply:
:pirate: arrrrrrr

Reply:
That error is only raised if the entire line was just a '>', the error after that one is raised if the line started with a '> '. For that error on line 168, since the ID is the only mandatory part and the description is optional, it seems most appropriate to raise an error saying the line is missing an ID if the line contains only '>'. For the error after that one on line 172, maybe the wording can be changed to say 'ID (or description)'  instead of just ID?

Reply:
I'm sorry I misread this branch (I wasn't thinking about the `>` still in `line`), you are correct. We can ignore this comment, the one on 172 is fine as is I think (because it is most likely a broken ID, not a description).